generate-schema.js

```diff
@@ -39,18 +44,27 @@ const defs = {
 for (let i=0; i < systems.length; i++) {
     const system = systems[i]
     let versionsEnum = []
-    for (let i=0; i < system.versions.length; i++) {
-        const ver = system.versions[i]
-        versionsEnum = versionsEnum.concat(ver)
+
+    if (system.versions !== undefined) {
+        for (let i = 0; i < system.versions.length; i++) {
+            const ver = system.versions[i]
+            versionsEnum = versionsEnum.concat(ver)
+        }
+    }
+
+    // Key on the distribution, except for Windows where this is irrelevant
+    key = system.distribution
```

Comment:
Just a minor nit pick: add a `let` declaration here?

Reply:
I updated the eslint config to reflect this in the future 👍